Bond/iOS/Bond+UIView.swift

```diff
@@ -71,4 +72,18 @@ extension UIView {
       return d
     }
   }
+    
+  public var dynUserInteractionEnabled: Dynamic<Bool> {
+    if let d: AnyObject = objc_getAssociatedObject(self, &userInteractionEnabledUIView) {
+      return (d as? Dynamic<Bool>)!
+    } else {
+      let d = InternalDynamic<Bool>(self.userInteractionEnabled)
+      let bond = Bond<Bool>() { [weak self] v in if let s = self { s.userInteractionEnabled = v } }
```

Comment:
@ivanmoskalev Thanks for the input, I created an issue for the proposed changes so that discussion can take place. I might take ownership of the issue and make the change in the near future.